### DISCUSSION TITLE
A separate posts overview for each tag

### DISCUSSION THREAD

**Discussion by Hariko50** [GetPublii/Publii#1911] [Help] [answered]
How do I create a separate post overview for each tag? It doesn't work with the post index link, or can I add the desired tag there?

**Comment by michaelfluegge** [comment, marked as answer]
It will be created automatically when you click on a tag. Try https://demo.getpublii.eu/themes/simple/v3/travel-guide-to-japan/ . Scroll down to the list of tags, choose one tag and a new page with a list of post related to the tag appear.

**Reply by michaelfluegge** [reply]
IF you want to do it on your own, start here https://getpublii.com/dev/create-post-list-from-specific-tags/.

**Reply by Hariko50** [reply]
Thank you, I now understand how to proceed.
